gnomad_qc/v4/sample_qc/hard_filters.py

```diff
@@ -51,16 +65,24 @@ def compute_sample_qc(n_partitions: int = 500, test: bool = False) -> hl.Table:
     vds = hl.vds.filter_intervals(
         vds, intervals=telomeres_and_centromeres.ht(), keep=False
     )
-
+    if n_alt_alleles_strata < 2:
+        raise ValueError("'n_alt_alleles_strata' must be greater than or equal to 2!")
+    elif n_alt_alleles_strata == 2:
+        strata = {
+            "bi_allelic": bi_allelic_expr(vds.variant_data),
+            "multi_allelic": ~bi_allelic_expr(vds.variant_data),
+        }
+    else:
+        strata = {
+            f"under_{n_alt_alleles_strata_name}_alt_alleles": vds.variant_data.n_unsplit_alleles
+            <= n_alt_alleles_strata,
+            f"{n_alt_alleles_strata_name}_or_more_alt_alleles": vds.variant_data.n_unsplit_alleles
+            > n_alt_alleles_strata,
+        }
```

Comment:
`n_unsplit_alleles` includes the reference allele.

So if split at 3 alt alleles (`n_alt_alleles_strata = 3`) the stratification will be:
  - under 3 alt alleles (under_three_alt_alleles)
    - same as under 4 total alleles (`n_unsplit_alleles` < 4)
    - same as less than or equal to 3 total alleles (`n_unsplit_alleles` <= 3)
  - 3 or more alt alleles (three_or_more_alt_alleles)
    - same as 4 or more total alleles (`n_unsplit_alleles` >= 4)
    - same as greater than 3 total alleles (`n_unsplit_alleles` > 3)

So it can be as is, or it can be:
```
        strata = {
            f"under_{n_alt_alleles_strata_name}_alt_alleles": (vds.variant_data.n_unsplit_alleles - 1)
            < n_alt_alleles_strata,
            f"{n_alt_alleles_strata_name}_or_more_alt_alleles": (vds.variant_data.n_unsplit_alleles - 1)
            >= n_alt_alleles_strata,
        }
```
or 
```
        strata = {
            f"under_{n_alt_alleles_strata_name}_alt_alleles": vds.variant_data.n_unsplit_alleles
            < (n_alt_alleles_strata + 1),
            f"{n_alt_alleles_strata_name}_or_more_alt_alleles": vds.variant_data.n_unsplit_alleles
            >=  (n_alt_alleles_strata + 1),
        }
```